strings/strings.json

```diff
@@ -2674,6 +2674,14 @@
     "zh-cn": "展开",
     "zh-tw": "展開"
   },
+  "CollapseFacet": {
+    "en": "Collapse {0} facet",
```

Comment:
I'm not sure I like the term `facet`here, since this is meant to be read to end users, and not "implementer".

Maybe `Collapse filter` / `Expand filter` ...?  🤔

Reply:
We should change it in other places then, there are a lot of strings containing facet:
ShowFacet
HideFacet
SaveFacetState
ClearFacetState
Facets
FacetTitle
I think Facet is fine when talking about the UI component
Facets apply filters ....
Or we can change the facet for filter in most these places
What do you think @fbeaudoincoveo  ?

Reply:
Mmm for consistency's sake, we'd probably want to update these occurrences to the equivalent word for "filter" in other languages too.

E.g., "Agrandir le filtre {0}"

Sounds pretty weird in French :)

I agree with JF: I believe using _facet_ is fine when referring to the UI component; if we're talking about selecting facet _values_ then we may want to consider using "filter" in some instances. That'd really be a case by case thing, though.

In short, I think we should leave it to "Collapse {0} facet".